### PR TITLE
Documentation and CHANGELOG.md entry for #2579.

### PR DESCRIPTION
**Pull request by pstibrany**
This PR adds CHANGELOG entry and documentation for PR #2579.

### CODE REVIEW

**Comment by pstibrany** [comment]
> Perfect! Just DCO & LGTM!

🤦

**Comment by pstibrany** [comment]
> @pstibrany One thing I have realized, we mentioned and documented config file to use it, however the flag itself is not documented and it is hidden (if it hasn't changed). This might create confusion. Should we also make it visible or at least mention it somewhere?

That's a good point. I will put it into documentation and changelog entry.

File: docs/components/store.md [hunk 1]
```diff
@@ -274,30 +274,44 @@ While the remaining settings are **optional**:
...
+metafile_exists_ttl: 2h
+metafile_doesnt_exist_ttl: 15m
+metafile_content_ttl: 24h
+metafile_max_size: 1MiB
 ```
 
 `backend_config` field for memcached supports all the same configuration as memcached for [index cache](#memcached-index-cache).
 
-`caching_config` is a configuration for chunks cache and supports the following optional settings:
+`caching_config` is a configuration for cache and supports the following optional settings for chunks caching:
```
Comment (GiedriusS) [inline docs/components/store.md:300]: But `caching_config` key doesn't exist, right? :thinking: So, I think this line should be reworded.

Comment (pstibrany) [reply docs/components/store.md]: You're correct. I've removed mention of `caching_config`.

File: docs/components/store.md [hunk 2]
```diff
@@ -274,31 +274,45 @@ While the remaining settings are **optional**:
...
 - `max_chunks_get_range_requests`: how many "get range" sub-requests may cache perform to fetch missing subranges.
 - `chunk_object_size_ttl`: how long to keep information about chunk file length in the cache.
 - `chunk_subrange_ttl`: how long to keep individual subranges in the cache.
 
-Note that chunks cache is an experimental feature, and these fields may be renamed or removed completely in the future.
+Following options are used for metadata caching (meta.json files, deletion mark files, iteration result):
+
+- `blocks_iter_ttl`: how long to cache result of iterating blocks.
+- `metafile_exists_ttl`: how long to cache information about whether meta.json or deletion mark file exists
+- `metafile_doesnt_exist_ttl`: how long to cache information about whether meta.json or deletion mark file doesn't exist
```
Comment (pracucci) [inline docs/components/store.md:311]:
```suggestion
- `metafile_doesnt_exist_ttl`: how long to cache information about whether meta.json or deletion mark file doesn't exist.
```

**Pull request merged**